compiler/src/codegen.rs

```diff
@@ -155,32 +155,34 @@ impl<'a> MethodGen<'a> {
                 method_name,
                 self.input(),
                 fq_grpc("CallOption"),
-                fq_grpc("UnaryCallHandler"),
+                fq_grpc("ClientUnaryReceiver"),
                 self.output())
     }
 
     fn client_streaming(&self, method_name: &str) -> String {
-        format!("{}(&self) -> {}<{}, {}>",
+        format!("{}(&self) -> ({}<{}>, {}<{}>)",
                 method_name,
-                fq_grpc("ClientStreamingCallHandler"),
+                fq_grpc("ClientCStreamSender"),
```

Comment:
First Client means it's client side interface. CStream means it's a sender for client streaming call.